source/getting_started/how-to.rst

```diff
@@ -0,0 +1,105 @@
+
+    Use the following command to build a live boot image:
+
+    .. code-block:: bash
+
+        $ make SPARSE_IMG=true KERNELFLINGER_SUPPORT_USB_STORAGE=true gptimage -j $(nproc)
+
+    Substitute *$(nproc)* to the appropriate value according to the CPU cores on your build server.
+
+    After a successful build you get an *out/target/product/* **[lunch_target]/[lunch_target]** *_gptimage.img* image. For example, an *out/target/product/cel_kbl/cel_kbl_gptimage.img* image is generated for the *cel_kbl* launch target.
+
+    This is an image of a whole GPT disk, with an image size of 14GB. To change the default image size, edit the mixins config file *device/intel/project-celadon/[luch_target]/mixins.spec* as following:
+
+    .. code-block:: none
+
+        gptbuild: true(size=14G,generate_craff=false)
+
+#. Flash the live boot image to a USB disk.
+
+    For Linux, use the ``dd`` tool to easily flash the image file to the USB disk. For example:
+
+    .. code-block:: bash
+
+        $ sudo dd if=cel_kbl_gptimage.img of=/dev/sdc bs=1M
+
+    .. caution::
+        You **MUST** change the */dev/sdc* to the real USB disk device.
+
+        Be sure all the buffered data is written to the USB disk before removing the USB disk (e.g. run the ``sync`` command in Linux).
+
+    For Windows, use the ``rufus`` tool to flash the image file to the USB disk.
+
+#. Boot from the USB disk to Android directly.
+
+    Plug in the USB disk and power on. Then, press a function key to enter the BIOS boot menu. Select **boot** from the USB disk. Now, you can boot from the USB disk to Android directly.
+
+    If you choose to save data to this Android device and you have not removed the USB disk, the data will be saved to the USB disk. You can then use the saved data on other computers.
+
+    .. note::
+        #. If you already installed Android in the internal storage, disable the internal storage if you are using USB live boot.
+        #. If you want to use one USB disk in two computers, make sure the lock status is same in each computer.
+
+How to dual boot |C| and Windows
+--------------------------------
+
+You may want to install Windows and |C| Anroid on one device, and select which OS to boot everytime the device is powering up. To achieve the dual boot, you need to install Windows and |C| on two storage devices seperately. There are two approaches:
+
+#. Use Kernel Flinger UEFI executables:
+
+    The Kernel Flinger UEFI installer does not prompt for the target storage device to install. You need to disconnect the Windows installed storage device first, connect the storage back to the device after installing |C|, and then select which OS to boot from the BIOS boot menu.
```

Comment:
The Kernelflinger UEFI installer does not prompt the target storage device to install. Disconnect the Windows installed storage device first, connect the storage back to the device after installing |C|, and then select which OS to boot from the BIOS boot menu.